cirq/ops/three_qubit_gates.py

```diff
@@ -161,14 +161,21 @@ class ThreeQubitDiagonalGate(gate_features.ThreeQubitGate):
     def __init__(self,
                  diag_angles_radians: List[Union[float, sympy.Basic]]) -> None:
         """
+        A 3 qubit gate whose off-diagonal elements are zero and who on-diagonal
+        elements are all phases.
 
         Args:
             diag_angles_radians: The list of angles on the diagonal in radians.
-
+                If these values are $(x0, x1, \ldots , x^7)$ then the unitary
```

Comment:
nit: Caret seems wrong here, you probably meant `x0, x1, \ldots, x7` or `x_0, x_1, \ldots, x_7`.

Reply:
fixed